inc/assets.php

```diff
@@ -106,7 +106,7 @@ function localize_admin_script( $handle ) {
 		$handle,
 		'bylineData',
 		[
-			'addAuthorLabel'         => __( 'Search for an author to add to the byline', 'byline-manager' ),
+			'addAuthorLabel'         => __( 'Search for a byline author', 'byline-manager' ),
```

Comment:
I'd be curious to know whether this change came out of a specific experience with users of the plugin?

To me, from the outside, the string previously expressed two common-ish concepts ("an author" and "the byline") but now expresses one new-ish concept ("a byline author"). Bylines might also have multiple components of which authors are only one (see #8), which is encompassed a little more cleanly with the current string, in my opinion. So, IMO, some precision is being lost here. 

(Same applies to the "Author Bylines" string earlier in the PR.)

Reply:
You're right that is confusing. The confusing part for clients was not really understanding the difference between the author and the byline. The author people a person and the byline being the container for housing authors. This has been updated to be "Search for an author" which is essentially what's happening here.

Part of the update was to reduce the line length too.